response_operations_ui/templates/create-message.html

```diff
@@ -0,0 +1,112 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+<style>
```

Comment:
These should be in their own .scss file

Reply:
This is temporary css until the styling is sorted out properly. This css shouldn't be necessary but we're still waiting on the UX people to hear how the css should be done.